### DISCUSSION TITLE
Trame design customization

### DISCUSSION THREAD

**Discussion by khumuunee** [Kitware/trame#136] [Q&A] [answered]
Hello,

Is it possible to change(customize) the design below?
![4](https://user-images.githubusercontent.com/76699571/199213079-77e3ac77-4801-4c21-b323-b87a46b516ed.png)

Thanks!

**Comment by jourdain** [comment, marked as answer]
Can you explain what you would like to do and see?

**Reply by khumuunee** [reply]
I would like to change the words to another language. 
For example, Connection close -> Холболт хаагдлаа; Loading -> Холбогдож байна ... etc. 
Also, I would like to reduce the size of the loader.
In fact,  they are for user's requirements.

**Reply by jourdain** [reply]
Those error feedback are coming from:
- https://github.com/Kitware/trame-client/blob/master/vue-app/src/components/Connect/script.js#L68
- https://github.com/Kitware/trame-client/blob/master/vue-app/src/components/Connect/script.js#L79 

If you want to change the scale, you can apply style/css on [that element](https://github.com/Kitware/trame-client/blob/master/vue-app/src/components/Connect/script.js#L142) 

As of right now, those are hard coded but if you want to make them more generic, PR are welcome.

**Reply by khumuunee** [reply]
I've done it.

Many thanks!
